docs/content/reference/dynamic-configuration/file.yaml

```diff
@@ -518,7 +518,21 @@ tls:
       defaultCertificate:
         certFile: foobar
         keyFile: foobar
+      defaultGeneratedCert:
+        resolver: foobar
+        domain:
+          main: foobar
+          sans:
+            - foobar
+            - foobar
```

Comment:
Should these also have the `example.org` / `foo.example.org` / `bar.example.org` nomenclature used in the docs?

Reply:
not the reference files